redis/src/aio/connection.rs

```diff
@@ -41,6 +43,10 @@ pub struct Connection<C = Pin<Box<dyn AsyncStream + Send + Sync>>> {
 
     // Flag indicating whether resp3 mode is enabled.
     resp3: bool,
+
+    push_manager: PushManager,
+
+    con_addr: Arc<String>,
```

Comment:
please add documentation for every field which isn't self-explanatory (like this one) explaining what it represents, and where is it used, potentially.